src/Pim/Bundle/CatalogBundle/Resources/config/query_builders.yml

```diff
@@ -120,7 +120,7 @@ services:
     pim_catalog.query.elasticsearch.filter.group:
         class: '%pim_catalog.query.elasticsearch.filter.group.class%'
         arguments:
-            - ['groups']
+            - ['groups', 'variant_group']
```

Comment:
That's wierd, it was not working like this in 1.7 https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/CatalogBundle/Resources/config/query_builders.yml#L106